backend/app/models/knowledge.py

```
@@ -17,6 +17,7 @@
     Boolean,
     Column,
     DateTime,
+    JSON,
```

Ruff (0.14.10)

Redefinition of unused `JSON` from line 15

Remove definition: `JSON`

(F811)